openapi-specs/schema.json

```diff
@@ -1044,6 +1044,37 @@
         }
       }
     },
+    "BackupListResponse": {
+      "description": "The definition of a backup create response body",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "description": "The ID of the backup. Must be URL-safe and work as a filesystem path, only lowercase, numbers, underscore, minus characters allowed.",
```

Comment:
Came comment on the backup id as the same PR :)

Reply:
will take care of it in follow up PR